lib/Migration/RegisterBinary.php

```diff
@@ -67,6 +67,11 @@ public function run(IOutput $output): void {
 			}
 		}
 
+		// Make all binaries executable
+		chmod($binaryDir . "matterbridge-" . self::VERSION . '-linux-arm64', 0755);
+		chmod($binaryDir . "matterbridge-" . self::VERSION . '-linux-64bit', 0755);
+		chmod($binaryDir . "matterbridge-" . self::VERSION . '-linux-32bit', 0755);
```

Comment:
```suggestion
		chmod($binaryPath , 0755);
```

We will only execute this anyway

Reply:
Oh, wait, it wouldn't work anyways unless the migration is rerun if the server is moved to another architecture. Didn't think about that.